style.css

```diff
@@ -0,0 +1,60 @@
+body {
+  margin: 0;
+  padding: 0;
+
+  background-image: linear-gradient(
+      115deg,
+      rgba(58, 58, 158, 0.8),
+      rgba(136, 136, 206, 0.7)
+    ),
+    url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+  
+  background-position: center;
+  height: 100%;
+  width: 100%;
+  background-size: cover;
+
+  top: 0;
+  bottom: 0;
+
+  text-align: center;
+  color: white;
+  background-color: #1b1b32;
+  
+
+}
+
+#parent {
+  align-items: center;
+}
+
+#form {
+  background-color: black;
+  margin: 100px 300px;
+}
+#form input {
+  width: 50%;
+  height: 20px;
+}
+
+#form select {
+    width: 50%;
+  }
+#form textarea {
+width: 50%;
+height: 200px;
+}
+
+#submit {
+    margin: 30px;
+    color: white;
+    background-color: green;
```

Comment:
For readability, add two spaces of indentation. Do not use the tab key

Reply:
- [ ] Orders again.. please make sure to only make polite suggestions.

Reply:
## OPTIONAL

- [ ] Good job explaining why this change was requested 👍🏿 
 
- [ ] You could also add emojis to your comments, they make the code review more human-friendly 👽🤖.